src/options/theory_options.toml

```diff
@@ -32,6 +32,24 @@ name   = "Theory Layer"
   default    = "true"
   help       = "condense values for functions in models rather than explicitly representing them"
 
+[[option]]
+  name       = "defaultFunctionValueMode"
+  category   = "expert"
+  long       = "default-function-value-mode=MODE"
+  type       = "DefaultFunctionValueMode"
+  default    = "FIRST"
+  help       = "mode for choosing default values for functions"
+  help_mode  = "Defines mode for choosing default values for functions."
+[[option.mode.FIRST]]
+  name = "first"
+  help = "The default value is equal to the value of the first application found of that function when applicable, or the first enumerated term otherwise."
```

Comment:
1. When is the "first application found" applicable? And is it deterministic? Will it still be deterministic after the stability stuff enter main?
2. In the enumerated case, according to which enumeration?

Reply:
For 1, this order in which terms are registered to the model, which can depend on assertion order.